lib/realtime/tenants.ex

```diff
@@ -104,6 +104,7 @@ defmodule Realtime.Tenants do
           end
         end)
 
+        Process.exit(db_conn, :kill)
```

Comment:
Not sure if db_conn will be restarted if the process is killed. Perhaps `Genserver.stop(db_conn)` might make sense here(https://stackoverflow.com/questions/44702375/how-to-disconnect-postgrex-connection/44703349#44703349).
To void error first check if pid is alive: https://hexdocs.pm/elixir/1.12/Process.html#alive?/1

That said I did see this pattern in other places in the code, so testing it in iex would be the best

Reply:
actually I will start switching to use GenServer.stop as we been bitten before by Process.exit due to a :normal vs :kill exit (didn't kill child processes)